matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/room/create/CreateLocalRoomTask.kt

```diff
@@ -77,6 +81,7 @@ internal class DefaultCreateLocalRoomTask @Inject constructor(
         @SessionDatabase private val realmConfiguration: RealmConfiguration,
         private val createRoomBodyBuilder: CreateRoomBodyBuilder,
         private val userService: UserService,
+        private val cryptoService: DefaultCryptoService,
```

Comment:
I think we should inject the interface `CryptoService` instead, no?

Reply:
Oh it is because the `onStateEvent` is not part of the service interface... Strange maybe we should move the method to the interface. @bmarty What do you think?

Reply:
Yes, I would not change it, I need it to register the local room in the crypto store in the same way as it is done for an existing room. I agree that it should be refactored, but I suppose that the intention was to not expose it in the interface.

Reply:
We should not inject service inside the SDK, and we should not inject implementations as well, but we have some exceptions...
The Service interface should contain only methods useful for the application, not the internal stuff.
So OK to keep it like this.